scripts/travis/deploy-linux-build.sh

```diff
@@ -64,19 +64,21 @@ echo "Deploy travis build on the server..."
 # Add server public key to known hosts.
 echo $DEPLOY_SSH_KEY >> $HOME/.ssh/known_hosts
 
-# Send build to server.
+# Remove previous build from server.
 export SSHPASS=$DEPLOY_PASSWORD
+sshpass -e ssh $DEPLOY_USER@$DEPLOY_URL rm -rf $DEPLOY_FOLDER
+
+# Send new build to the server.
 sshpass -e rsync \
     --recursive \
     --archive \
     --compress \
     --stats \
     --no-perms --no-owner --no-group \
-    --include="build_report.txt" \
-    --include="sanbox/" \
-    --include="scripts/" \
-    --include="prebuilt-linux-deps/" \
-    --delete-after \
+    build_report.txt \
+    prebuilt-linux-deps \
+    sandbox \
+    scripts \
     ./ \
```

Comment:
If you name every folder and file explicitely, you have to remove the base dir, otherwise it will be copied over as well.